Inputs/config.js

```diff
@@ -20,104 +20,134 @@ const accountSetting = {
+  // "MATIC/USDT",
+  // "THETA/USDT",
+  // "HNT/USDT",
+  // "DOT/USDT",
+  // "YFI/USDT",
+  // "LUNA/USDT",
+  // "UNI/USDT",
+  // "AVAX/USDT",
+  // "AAVE/USDT",
+  // "QNT/USDT",
+  // "AXS/USDT",
+  // "FIL/USDT",
+  // "ETH/USDC",
+  // "BNB/USDC",
+  // "BTC/USDC",
+  // "XRP/USDC",
+  // "ADA/USDC",
+  // "BAT/USDC",
+  // "LINK/USDC",
+  // "SOL/USDC",
+  // "TRX/USDC"
 ]
 
-const symbolsForMarkPrice = [
-  'USDC/USDT',
-  'ALGO/USDT',
+let symbolsForMarkPrice = [
   "ETH/USDT",
-  "BNB/USDT",
-  "BTC/USDT",
-  "XRP/USDT",
-  "ADA/USDT",
-  "TRX/USDT",
-  "BAT/USDT",
-  "LINK/USDT",
-  "MANA/USDT",
-  "SOL/USDT",
-  "MATIC/USDT",
-  "THETA/USDT",
-  "HNT/USDT",
-  "DOT/USDT",
-  "YFI/USDT",
-  "LUNA/USDT",
-  "UNI/USDT",
-  "AVAX/USDT",
-  "AAVE/USDT",
-  "QNT/USDT",
-  "AXS/USDT",
-  "FIL/USDT"
+  // "BNB/USDT",
+  "BTC/USDT"
+  // 'USDC/USDT',
+  // 'ALGO/USDT',
+  // "ETH/USDT",
+  // "BNB/USDT",
+  // "BTC/USDT",
+  // "XRP/USDT",
+  // "ADA/USDT",
+  // "TRX/USDT",
+  // "BAT/USDT",
+  // "LINK/USDT",
+  // "MANA/USDT",
+  // "SOL/USDT",
+  // "MATIC/USDT",
+  // "THETA/USDT",
+  // "HNT/USDT",
+  // "DOT/USDT",
+  // "YFI/USDT",
+  // "LUNA/USDT",
+  // "UNI/USDT",
+  // "AVAX/USDT",
+  // "AAVE/USDT",
+  // "QNT/USDT",
+  // "AXS/USDT",
+  // "FIL/USDT"
 ]
 
-const symbolsForSettlementPrice = [
-  'USDC/USDT',
-  'ALGO/USDT',
+let symbolsForSettlementPrice = [
   "ETH/USDT",
-  "BNB/USDT",
-  "BTC/USDT",
-  "XRP/USDT",
-  "ADA/USDT",
-  "TRX/USDT",
-  "BAT/USDT",
-  "LINK/USDT",
-  "MANA/USDT",
-  "SOL/USDT",
-  "MATIC/USDT",
-  "THETA/USDT",
-  "HNT/USDT",
-  "DOT/USDT",
-  "YFI/USDT",
-  "LUNA/USDT",
-  "UNI/USDT",
-  "AVAX/USDT",
-  "AAVE/USDT",
-  "QNT/USDT",
-  "AXS/USDT",
-  "FIL/USDT"
+  // "BNB/USDT",
+  "BTC/USDT"
+  // 'USDC/USDT',
+  // 'ALGO/USDT',
+  // "ETH/USDT",
+  // "BNB/USDT",
+  // "BTC/USDT",
+  // "XRP/USDT",
+  // "ADA/USDT",
+  // "TRX/USDT",
+  // "BAT/USDT",
+  // "LINK/USDT",
+  // "MANA/USDT",
+  // "SOL/USDT",
+  // "MATIC/USDT",
+  // "THETA/USDT",
+  // "HNT/USDT",
+  // "DOT/USDT",
+  // "YFI/USDT",
+  // "LUNA/USDT",
+  // "UNI/USDT",
+  // "AVAX/USDT",
+  // "AAVE/USDT",
+  // "QNT/USDT",
+  // "AXS/USDT",
+  // "FIL/USDT"
 ]
 
 const symbolLastUpdate = {}
 
+const fetchActiveSymbol = async(client)=>{
+  const query = `select distinct symbol from dbt_traderdao.activeasset`;
+  const res = await client.query(query);
+  const newAsset = res.rows;
+  // await client.end()
+  // console.log('newAsset', newAsset)
+  const tradingQuotation = newAsset.map((symbol)=>{
+    return symbol.symbol
+  }).filter((symbol)=>{
+    let regex = /LD/i
+    let regex2 = /ETHW/i
+    return !(symbol.match(regex)) && !(symbol.match(regex2))
+  })
+  const quotation = tradingQuotation.filter((symbol)=>{
+    let regex = /USDT/i
+    return symbol.match(regex)
+  })
+  // console.log(quotation)
+  symbolsForMarkPrice = quotation;
+  symbolsForSettlementPrice = quotation;
+  symbols = tradingQuotation;
```

Comment:
Instead of changing variables outside the function scope directly, I think it's better to keep thing inside the function. 

That is, you can return those values with an object
```
return {
  symbolsForMarkPrice: quotation,
  symbolsForSettlementPrice: quotation,
  symbols: tradingQuotation
}
```
When you need these values, you can write
```
const { symbolsForMarkPrice, symbolsForSettlementPrice, symbols } =  await fetchActiveSymbol(client);
```